tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -9126,6 +9138,12 @@ TEST_F(EnergyPlusFixture, UnitarySystemModel_MultispeedDXCoilSizing)
     EXPECT_NEAR(1.0 / 10, thisSys->m_MSHeatingSpeedRatio[1] / thisSys->m_MSHeatingSpeedRatio[10], 0.00001);
 
     EXPECT_FALSE(thisSys->m_MyFanFlag); // fan speed variables have been set
+
+    // test system no load air volume flow rate
+    Real64 result1_expected_NoLoadAirVolFlow =
+        state->dataUnitarySystems->designSpecMSHP[0].noLoadAirFlowRateRatio * std::min(thisSys->m_MaxCoolAirVolFlow, thisSys->m_MaxHeatAirVolFlow);
+    EXPECT_EQ(1.0, state->dataUnitarySystems->designSpecMSHP[0].noLoadAirFlowRateRatio);
+    EXPECT_EQ(result1_expected_NoLoadAirVolFlow, thisSys->m_MaxNoCoolHeatAirVolFlow);
 }
 
```

Comment:
Modified another existing unit test to demonstrate the fix for a different multi-speed DX cooling coil type and different `No Load Supply Air Flow Rate Ratio` value.